src/main/java/persistence/entity/EntityKey.java

```diff
@@ -0,0 +1,36 @@
+package persistence.entity;
+
+import persistence.model.EntityPrimaryKey;
+
+import java.util.Objects;
+
+public class EntityKey {
+    private final Class<?> entityClass;
+
+    private final EntityPrimaryKey pk;
```

Comment:
EntityKey 를 도출해주셨네요 💯 
불필요한 공백을 없애보면 어떨까요?
```suggestion
public class EntityKey {
    private final Class<?> entityClass;
    private final EntityPrimaryKey pk;
```